### PR TITLE
Sidetrack cleanup and change restoration

### PR DESCRIPTION
**Pull request by bkarnow**
Addresses both https://phabricator.endlessm.com/T30478 and the very last parts of https://phabricator.endlessm.com/T30359

There were some changes in the initial solution to T30359 that didn't make it in after the decision to disable to level selectors. This restores those changes, normalizes the formatting, and streamlines the quest. Also, this fixes the last remaining issues from T30359, as well as the sync issues in T30478, and the attract issue in T30480.

Flow edits:
- Add easter-egg early completion detection in level 24
- Streamline level 26/27
- Move level selector disable earlier, otherwise it flashes after level 1 is completed
- Remove level22_reorder
- Condense level23 / level23_2 to get the player hacking faster
- Partially addressed T30478 by adding another level-end check
- Add flip detectors to fix the rest of T30478
- Restore badge grant to level 24
- Add a redundant disabling of the FtH attract in level 25, to prevent it from flashing for the rest of the quest (https://phabricator.endlessm.com/T30480)

Text edits:
- Reword some lines to be simpler
- Add lines in 26_2 to make it more clear that the errors are part of the game, not an actual mistake.
- Reword some dialogues to remove characters and streamline nodes

Code formatting:
- Fix code newline spacing (1 after in-level node, 2 between levels)
- Fix spacing after initial "-"
- Fix bad space in ST1 Badge grant and "currentLevel-1"
- Condense flip detectors
- Remove "cont_" from all node names for clearer reading

https://phabricator.endlessm.com/T30478
https://phabricator.endlessm.com/T30359
https://phabricator.endlessm.com/T30480

### CODE REVIEW

File: src/ui/quests/sidetrack-2-quest.ink [hunk 1]
```diff
@@ -47,105 +27,103 @@
...
-* ❯
-# character: faber
-- Wow, looks like this level has 2 errors! Time to <b>flip</b> the app and get to the <b>Instructions</b> again!
+- There are still plenty of challenges ahead, though. Let's keep going, <b>flip</b> the app and hack the <b>Instructions</b> again!
 * [(wait for: flipped)] Flipped!
 -> level24_3
-* [(wait for: currentLevel is 25)] Level {currentLevel -1} Complete!
+* [(wait for: currentLevel is 25)] #
+# character: saniel
+- Gibbering heffalumps! That's quite a gambit, {get_user_name()}! I think I might say... how does it go... that "you're the man now, dog!"
```
[the remark this answers is not in the document]
Comment (bkarnow) [reply src/ui/quests/sidetrack-2-quest.ink:124]: This level is beatable without hacking - but it requires you to be very clever and exploit the properties of `error` instructions. If the player is clever enough to figure that out, they deserve an acknowledgement!

File: src/ui/quests/sidetrack-2-quest.ink [hunk 2]
```diff
@@ -201,85 +180,68 @@
...
 -> level26_2
 
 
 === level26_2 ===
-- Now you're getting it! Head to the <b>Instructions</b> again by <b>flipping</b> Sidetrack!
+- Yeah, almost like... it's intentional... :D
+# character: felix
+- <b><i>m*::>^_^<::&rr0w>!</i></b>
+- You said it, Felix! Hey {get_user_name()}, you know the drill now! <b>Flip</b> and hack!
+{not flipped: -> level26_3}
```
[the remark this answers is not in the document]
Comment (bkarnow) [reply src/ui/quests/sidetrack-2-quest.ink:220]: I think so, yes, verifying.

File: src/ui/quests/sidetrack-2-quest.ink (hunk 1 above)
[the remark this answers is not in the document]
Comment (manuq) [reply src/ui/quests/sidetrack-2-quest.ink:124]: Gotcha! Thanks for explaining. Makes sense. Just let the quest say "level complete" and then acknowledge the player. Something like:
```
* [(wait for: flipped)] Flipped!
  -> level24_3
* [(wait for: currentLevel is 25)] Level {currentLevel-1} Complete!
  Gibbering heffalumps! ... # character: saniel
  -> level25

```
Also, see how indentation helps to read the lines nested inside choices. I indented everything with 2 spaces in the Spanish version.

File: src/ui/quests/sidetrack-1-quest.ink [hunk 3]
```diff
@@ -42,40 +43,35 @@
...
 # character: estelle
--Watch how the bugbots move and where they go, and think about the spaces they open up when they move.
+- Watch how the bugbots move and where they go, and think about the spaces they open up when they move.
 -> mid_level_check(8) -> level7_3
 
 === level7_3 ===
-# character: estelle
--These bugbots <b>only move down</b>, never left or right, so that might let you find a... bug... in their behavior. ^_^
-Oh yeah, when they walk off the bottom, they reappear at the top! So, keep your eye out for that!
+# character: riley
```
[the remark this answers is not in the document]
Comment (bkarnow) [reply src/ui/quests/sidetrack-1-quest.ink:72]: Missed that one, thank you!

**Comment by bkarnow** [comment]
Feedback addressed!

File: src/ui/quests/sidetrack-2-quest.ink [hunk 4]
```diff
@@ -98,37 +98,37 @@
...
 === level23_flip ===
 {not flipped: -> level23_play}
-~ attractFTH = 1
 - Now <b>flip</b> back to the front of Sidetrack.
+~ attractFTH = 1
 * [(wait for: flipped)]  Flipped back!
+~ attractFTH = 0
 -> level23_play
 * [(wait for: currentLevel is 24)] Level {currentLevel-1} Complete!
+~ attractFTH = 0
```
Comment (manuq) [inline src/ui/quests/sidetrack-2-quest.ink]: :+1:

File: src/ui/quests/sidetrack-2-quest.ink [hunk 5]
```diff
@@ -208,24 +207,18 @@
 - I don't think these errors are a mistake! Every instruction in this set is wrong, and statistically, that's very unlikely.
 * ❯
 -> level26_2
+* [(wait for: flipped)] Flipped!
+-> level26_2
 * [(wait for: currentLevel is 27)] Level {currentLevel-1} Complete!
 -> level26_2
```
Comment (manuq) [inline src/ui/quests/sidetrack-2-quest.ink]: Suggestion: here all choices end doing the same. This is why gathers exist! You can rewrite this:

```
* ❯
-> level26_2
* [(wait for: flipped)] Flipped!
-> level26_2
* [(wait for: currentLevel is 27)] Level {currentLevel-1} Complete!
-> level26_2
```
To this:

```
* ❯
* [(wait for: flipped)] Flipped!
* [(wait for: currentLevel is 27)] Level {currentLevel-1} Complete!
- -> level26_2
```

**Pull request merged**